validator/lib/readTileset.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+var fs = require('fs-extra');
+var Promise = require('bluebird');
+
+var fsReadFile = Promise.promisify(fs.readJson);
+
+function readTileset(filePath) {
+    return fsReadFile(filePath)
```

Comment:
Oh sorry, I didn't realize you promisified the readJson function based on the naming. Still it's a good idea to follow these steps.